### ISSUE TITLE
Can't configure multible ports

### ISSUE THREAD

**Issue by bmack94** [bodgit/puppet-openssh#2]
According to: https://github.com/bodgit/puppet-openssh/blob/master/manifests/server.pp#L229 i can configure multiple ports as an array
But in https://github.com/bodgit/puppet-openssh/blob/master/templates/server/config.erb#L216 it won't iterate over the given array.

This could be solved by replacing it with this erb snippet:
```
<% if @port -%>
<% @port.each do |val| -%>
Port <%= val %>
<% end -%>
<% end -%>
```

**Comment by bodgit** [comment]
D'oh! Good catch!

**Comment by bodgit** [comment]
Should be fixed in v1.0.3 which will be uploaded to the forge shortly. Thanks for the bug report!
